site/content/en/docs/getting-started-deploy.md

```diff
@@ -1,9 +1,10 @@
 ---
-title: "prow/getting_started_deploy.md"
+title: "Deploying Prow"
```

Comment:
We might add some categories for these pages.
For now these pages are just placed in the top-level directory of the Prow docs codebase.
![image](https://user-images.githubusercontent.com/46767780/203288782-151d2b8a-dd53-4a32-b694-9330bf335881.png)

E.g., K8s docs: (https://github.com/kubernetes/website/tree/main/content/en/docs)
- `concepts`
- `contribute`
- `reference`
- `setup`
- `tasks`
- `tutorials`